app/models/reaction.rb

```diff
@@ -2,7 +2,5 @@ class Reaction < ApplicationRecord
   belongs_to :solution
   belongs_to :user
 
-  enum emotion: [ :like, :love, :genius, :legacy ]
-
-  scope :with_comments, -> { where("comment != '' AND comment IS NOT NULL") }
+  enum emotion: [ :like, :love, :genius ]
```

Comment:
These come with convenience methods like `emotion.like?` Can't you use those in the views too? (And then you can meta-program  the icon into it 🤓 )

Reply:
Yeah. Nice. We're getting rid of the different type of reactions altogether now FYI.